odoo/addons/base/tests/test_mail.py

```diff
@@ -86,6 +86,46 @@ def test_html(self):
         for attr in ['javascript']:
             self.assertNotIn(attr, sanitized_html, 'html_sanitize did not remove enough unwanted attributes')
 
+    def test_outlook_mail_sanitize(self):
+        case = """<div class="WordSection1">
+<p class="MsoNormal">Here is a test mail<o:p></o:p></p>
+<p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+<p class="MsoNormal">With a break line<o:p></o:p></p>
+<p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+<p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+<p class="MsoNormal">Then two<o:p></o:p></p>
+<p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+<div>
+<div style="border:none;border-top:solid #E1E1E1 1.0pt;padding:3.0pt 0in 0in 0in">
+<p class="MsoNormal"><b>From:</b> Mitchell Admin &lt;dej@gmail.com&gt;
+<br>
+<b>Sent:</b> Monday, November 20, 2023 8:34 AM<br>
+<b>To:</b> Eteil DJOUMATCHOUA &lt;dej@gmail.com&gt;<br>
+<b>Subject:</b> test (#23)<o:p></o:p></p>
+</div>
+</div>"""
+
+        expected = """<div class="WordSection1">
+<p class="MsoNormal">Here is a test mail</p>
+<p class="MsoNormal"> </p>
+<p class="MsoNormal">With a break line</p>
+<p class="MsoNormal"> </p>
+<p class="MsoNormal"> </p>
+<p class="MsoNormal">Then two</p>
+<p class="MsoNormal"> </p>
```

Comment:
Isn't it the inverse (2 break lines, then one) ?

Reply:
It's the break lines above XD